src/newSeed/stage8.html

```diff
@@ -9,7 +9,7 @@
 
         <p>You may inspect the Seed configuration, stored in the JSON format in IPFS, here:
           <a href="${ipfsService.getIpfsUrl(wizardState.seedHash)}" target="_blank" rel="noopener noreferrer">${ipfsService.getIpfsUrl(wizardState.seedHash)}</a>.
-          Please save this URL.
+          <b>Please save this URL</b>.
```

Comment:
As we have discussed before, we won't use tags such as `<i>`, `<b>` to control styling.  We use classes.